src/plugins/controlbox/styles/_controlbox.scss

```diff
@@ -53,6 +53,7 @@
     #controlbox {
         order: -1;
         color: var(--controlbox-text-color);
+        overflow-y:none;
```

Comment:
Is `none` a valid value?

It's not shown here as one: https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-y

What's the effect of this? Does it disable vertical scrolling in the controlbox?
If so, I'm not convinced that this is the right solution, since the controlbox might go off-screen on certain displays.

Did you check what happens in fullscreen mode? Might be that this bug should only be fixed for overlayed view mode, in which case you need to put any CSS changes under the following:

```
.conversejs {
    converse-chats {
        &.converse-overlayed {
            #controlbox {
```

Reply:
yeah it's valid value it's work like hidden property but i can changes to the hidden for convinence,

i found a one more thing while checking the chat box there is a dragabble property, and we can resize the chat box but we should stop the resize property to a specific height so we can set the flyout min-height to 500px so that when user try to resize this to 400px it won't be an we can't see any scrollbar this is my approch to solve this bug.

Reply:
> i can changes to the hidden for convinence

Yes, `hidden` would be better.

Reply:
I've thought about this some more, and I don't think this is the right solution.

You're simply disabling scrolling, which means that if the contents would overflow, it would be invisible and the user cannot scroll to see it.

Also, since I don't know where those icons reported in #3133 come from, it's not clear to me what needs to be fixed, how it should be done or whether there really is anything to fix.